lib/manifoldco_signature/signature.rb

```diff
@@ -0,0 +1,56 @@
+require 'base64url'
+require 'rbnacl/libsodium'
+
+module ManifoldcoSignature
+
+  class Signature
+
+    def initialize(req)
+      sig = req.env['HTTP_X_SIGNATURE'].split
```

Comment:
As an aside though, show we check the length of the resulting `sig` to be `3`, or only rely on the base64url decode to throw?